joint_trajectory_controller/test_robot_components.xml

```diff
@@ -1,5 +1,5 @@
 <library path="test_jtc_robot_hardware_components">
-  <class name="test_robot_hardware/TestPosVelSingleJointActuator" type="test_robot_hardware::TestPosVelSingleJointActuator" base_class_type="hardware_interface::ActuatorInterface">
+  <class name="ros2_control_test_assets/TestPosVelSingleJointActuator" type="ros2_control_test_assets::TestPosVelSingleJointActuator" base_class_type="hardware_interface::ActuatorInterface">
```

Comment:
I don't think this file is doing anything. The mentioned component doesn't exist in `ros2_control_test_assets` and even if it would, it shouldn't be declared here. Did I miss something?

Reply:
Oh, this is some rest from porting. I just did mas-rename so this got also renamed. I can remove it.